src/constants/routing.ts

```diff
@@ -161,6 +161,8 @@ export const COMMON_BASES: ChainCurrencyList = {
   ],
   [ChainId.OPTIMISM]: [nativeOnChain(ChainId.OPTIMISM), OP, DAI_OPTIMISM, USDC_OPTIMISM, USDT_OPTIMISM, WBTC_OPTIMISM],
   [ChainId.OPTIMISM_GOERLI]: [nativeOnChain(ChainId.OPTIMISM_GOERLI)],
+  [ChainId.BASE]: [nativeOnChain(ChainId.BASE), WRAPPED_NATIVE_CURRENCY[ChainId.BASE] as Token],
+  [ChainId.BASE_GOERLI]: [nativeOnChain(ChainId.BASE_GOERLI), WRAPPED_NATIVE_CURRENCY[ChainId.BASE_GOERLI] as Token],
```

Comment:
I'm torn on this. They're the same address but this might be confusing to a dev picking this up if they didn't know Base was built on OP. I'd either add the duplicate `WRAPPED_NATIVE_CURRENCY` entry for Base and Base_Goerli, or add comments. Preferably the former.

Reply:
i'm confused by this comment - `WRAPPED_NATIVE_CURRENCY` has entries for the base networks